src/plone/restapi/serializer/vocabularies.py

```diff
@@ -43,8 +43,10 @@ def __call__(self, vocabulary_id):
         term = self.context
         token = term.token
         title = term.title if ITitledTokenizedTerm.providedBy(term) else token
+        value = term.value
         return {
             '@id': '{}/{}'.format(vocabulary_id, token),
             'token': token,
-            'title': title
+            'title': title,
+            'value': value
```

Comment:
I think it would be prudent to run `value` through the [`json_compatible()`](https://github.com/plone/plone.restapi/blob/1d4d683704f5f4b319e0bff111dcdb5984a8d308/src/plone/restapi/serializer/converters.py#L54-L72) helper function. This will ensure that any value is serialized the same way to JSON like it is in other places in `plone.restapi`.

Reply:
Amended. Thank you.